src/views/shortcodes/my-attendance-list.php

```diff
@@ -6,22 +6,41 @@
  *
  *     [your-theme]/tribe-events/shortcodes/my-attendance-list.php
  *
- * @version 4.8.2
+ * @since   4.8.2
+ * @since   TBD Removed target="_blank" from links, added direct link to each post's "My Tickets" view.
+ *
+ * @version TBD
  *
  * @var array $event_ids
  */
+
+$view = Tribe__Tickets__Tickets_View::instance();
 ?>
 
 <ul class="tribe-tickets my-attendance-list">
-	<?php foreach ( $event_ids as $id ): ?>
+	<?php
+	foreach ( $event_ids as $id ):
+		$is_event = function_exists( 'tribe_is_event' ) ? tribe_is_event( $id ) : false;
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 15 spaces but found 1 space

<sub>hash: 8dd56ba9346ec5030a5371f29e6d0a8d</sub>